worker/dbaccessor/worker.go

```diff
@@ -371,6 +403,42 @@ func (w *dbWorker) GetDB(namespace string) (coredatabase.TrackedDB, error) {
 	return tracked.(coredatabase.TrackedDB), nil
 }
 
+// DeleteDB deletes the dqlite-backed database that contains the data for
+// the specified namespace.
+// There are currently a set of limitations on the namespaces that can be
+// deleted:
+//   - It's not possible to delete the controller database.
+//   - It currently doesn't support the actual deletion of the database
```

Comment:
Technically, someone holding a `TrackedDB` for a model that gets deleted can still happily use it until we can actually remove the DB right?

Reply:
If they hold a `TrackedDB` and we stop and remove the worker, their db will be closed. All in-play transactions/queries will be closed and should/will cause the watchers et al to shut down.